Networking/TCPSocket.cc

```diff
@@ -622,4 +620,16 @@ namespace litecore { namespace net {
             setError(NetworkDomain, mbedToNetworkErrCode(err), slice(msgbuf));
         }
     }
+
+
+    bool TCPSocket::checkReadWriteStreamError() {
```

Comment:
The name of this function seems backwards -- it returns true if there's _not_ an error. I'd prefer that the return value be flipped.

Reply:
I just merged the PR so I will submit a new PR to fix this.